src/hotel_reservations/mlflow_processor.py

```
@@ -1,8 +1,17 @@
+import hashlib
+import random
+import time
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
 import mlflow
 import pandas as pd
+import requests
 from databricks import feature_engineering
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.catalog import *
```

Ruff

`from databricks.sdk.service.catalog import *` used; unable to detect undefined names

(F403)